jdi-light-mobile-tests/src/main/java/nativeapp/ios/contacts/AddNewContactPage.java

```diff
@@ -0,0 +1,12 @@
+package nativeapp.ios.contacts;
+
+import com.epam.jdi.light.mobile.elements.base.MobileAppUIElement;
+import com.epam.jdi.light.mobile.elements.common.ios.app.SystemButton;
+import com.epam.jdi.light.mobile.elements.pageobjects.annotations.MobileFindBy;
+
+public class AddNewContactPage {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'AddNewContactPage' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5531163)